src/sql/inspect.py

```diff
@@ -268,39 +376,94 @@ def __init__(self, table_name, schema=None) -> None:
                 # We ignore the cell stats for such case.
                 pass
 
+            table_stats[column] = _assign_column_specific_stats(
+                table_stats[column], is_numeric
+            )
+
         self._table = PrettyTable()
         self._table.field_names = [" "] + list(table_stats.keys())
 
-        rows = list(columns_to_include_in_report)
-        rows.sort(reverse=True)
-        for row in rows:
-            values = [row]
-            for column in table_stats:
-                if row in table_stats[column]:
-                    value = table_stats[column][row]
-                else:
-                    value = ""
-                value = util.convert_to_scientific(value)
-                values.append(value)
+        custom_order = [
+            "count",
+            "unique",
+            "top",
+            "freq",
+            "mean",
+            "std",
+            "min",
+            "25%",
+            "50%",
+            "75%",
+            "max",
+        ]
+
+        for row in custom_order:
+            if row.lower() in [r.lower() for r in columns_to_include_in_report]:
+                values = [row]
+                for column in table_stats:
+                    if row in table_stats[column]:
+                        value = table_stats[column][row]
+                    else:
+                        value = ""
+                    # value = util.convert_to_scientific(value)
+                    values.append(value)
+
+                self._table.add_row(values)
+
+        column_styles = _generate_column_styles(columns_with_styles)
+
+        if message_check:
+            message_content = _generate_message(columns_with_styles, list(columns))
+            warning_background = "#FFFFCC"
+            warning_title = "Warning: "
+        else:
+            message_content = ""
+            warning_background = "white"
+            warning_title = ""
 
-            self._table.add_row(values)
+        database = Connection.current.url
+        if "duckdb" in database:
```

Comment:
Can we utilize this method here? Also, we don't need to mention `only available with dubckdb`. Just mention that these are not available in `{db_driver}`
https://github.com/ploomber/jupysql/blob/a5790a62759310d37abb9d0b6354bae3cdc74a0a/src/sql/util.py#L246

Reply:
Can we add some integration tests here as well?

Reply:
Sure will add them.

Reply:
Made the changes